unix/povray.1

```diff
@@ -1,49 +1,41 @@
-.TH POV-Ray 1 "August 2013" "POV\-Team" "Version 3.7" \" -*- nroff -*-
+.TH POV-Ray 1 "December 2016" "POV\-Team" "Version 3.71" \" -*- nroff -*-
```

Comment:
I was aiming for a man page which matched the 3.7.1 master branch if someone did a 'make install' after a build. I added -+CC added to the flags for example.  

I agree the previous should have been 3.7.0. 

Do you want something more generic? Only documentation correct for all 3.7.0 and 3.7.1 ?